PyInstaller/depend/bytecode.py

```diff
@@ -64,6 +64,33 @@ def bytecode_regex(pattern: bytes, flags=re.VERBOSE | re.DOTALL):
     return re.compile(pattern, flags=flags)
 
 
+def finditer(pattern: re.Pattern, string):
+    """Call ``pattern.finditer(string)`` but remove any matches beginning on an
+    odd byte. i.e. match.start() is not a multiple of 2.
+
+    This should be used to avoid false positive matches where a bytecode pair's
+    argument is mistaken for an opcode.
+    """
+    matches = pattern.finditer(string)
+    while True:
+        for match in matches:
+            if match.start() % 2 == 0:
+                # All is good. This match starts on an OPCODE.
+                yield match
+            else:
+                # This match has started on an odd byte meaning that it's a
+                # false positive and should be skipped. There is a very slim
+                # chance that a genuine match overlaps this one and, because
+                # re.finditer() doesn't allow overlapping matches, it would be
+                # lost. To avoid that, restart the regex scan starting at the
+                # next even byte.
+                matches = pattern.finditer(string, match.start() + 1)
+                break
+            yield match
```

Comment:
This second yield will result in duplicated matches, will it not?

Reply:
Yes, I reversed the order of that `if/else` statement and left that in by accident. It's what CI was complaining about.